README.md

```diff
@@ -1,9 +1,10 @@
 # Auto Updater #
-**Contributors:** [1and1](https://profiles.wordpress.org/1and1), [markoheijnen](https://profiles.wordpress.org/markoheijnen), [pfefferle](https://profiles.wordpress.org/pfefferle), [gdespoulain](https://profiles.wordpress.org/gdespoulain)  
+**Contributors:** [1and1](https://profiles.wordpress.org/1and1), [ionos](https://profiles.wordpress.org/ionos), [markoheijnen](https://profiles.wordpress.org/markoheijnen), [pfefferle](https://profiles.wordpress.org/pfefferle), [gdespoulain](https://profiles.wordpress.org/gdespoulain)  
```

Comment:
Why still 1and1 in here?

Reply:
it is the old account, that might still be linked from various sources. If we remove the plugin from the account, some might be confused.